jupyter-extension/src/plugins/mount/components/Datum/Datum.tsx

```diff
@@ -114,12 +114,12 @@ const Datum: React.FC<DatumProps> = ({
           style={{display: 'flex'}}
         >
           <button
-            data-testid="Datum__mountDatums"
+            data-testid="Datum__loadDatums"
```

Comment:
This was not necessary to fix the tests, but it seemed like a low effort change to be consistent with naming so I handled it while I was here.